apps/ledger-live-desktop/src/renderer/families/internet_computer/ManageNeuronFlowModal/steps/FollowSelectTopics.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useCallback } from "react";
+import { KNOWN_TOPICS } from "@ledgerhq/live-common/families/internet_computer/consts";
+import Text from "~/renderer/components/Text";
+import Box from "~/renderer/components/Box";
+import WarnBox from "~/renderer/components/WarnBox";
+import CollapsibleCard from "~/renderer/components/CollapsibleCard";
+import { StepProps } from "../types";
+import { useTranslation } from "react-i18next";
+import { Trans } from "react-i18next";
+import Button from "~/renderer/components/Button";
+
+export function StepFollowSelectTopics({ setFollowTopic, transitionTo }: StepProps) {
+  const { t } = useTranslation();
+  const onClickSelectTopic = useCallback(
+    (topic: keyof typeof KNOWN_TOPICS) => {
+      setFollowTopic(topic);
+      transitionTo("selectFollowees");
+    },
+    [setFollowTopic, transitionTo],
+  );
+
+  return (
+    <Box>
+      <WarnBox>
+        <Trans i18nKey="internetComputer.manageNeuronFlow.selectTopics.warnbox" />
+      </WarnBox>
+      <Box>
+        <Text ff="Inter|SemiBold" fontSize={14} mb={10}>
+          <Trans i18nKey="internetComputer.manageNeuronFlow.selectTopics.listOfTopics" />
+        </Text>
+        <Box style={{ gap: 10 }}>
+          {Object.entries(KNOWN_TOPICS).map(([key]) => (
```

Comment:
`KNOWN_TOPICS` is a constant, so maybe create the array once outside the component.

```
const topicsList = Object.entries(KNOWN_TOPICS)
```

Reply:
fixed!